runtime/mandala/src/benchmarking/prices.rs

```diff
@@ -16,28 +16,26 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{CurrencyId, GetStableCurrencyId, Origin, Prices, Runtime};
+use crate::{Origin, Prices, Runtime};
 
-use super::utils::{dollar, feed_price};
+use super::utils::{dollar, feed_price, STABLECOIN};
 use frame_system::RawOrigin;
 use orml_benchmarking::runtime_benchmarks;
 use sp_std::vec;
 
-const STAKING: CurrencyId = GetStableCurrencyId::get();
-
 runtime_benchmarks! {
 	{ Runtime, module_prices }
 
 	lock_price {
 		// feed price
-		feed_price(vec![(STAKING, dollar(STAKING).into())])?;
-	}: _(RawOrigin::Root, STAKING)
+		feed_price(vec![(STABLECOIN, dollar(STABLECOIN).into())])?;
```

Comment:
why change to Stablecoins? we have some special handle logic for Stablecoins in prices module so better to still use staking here

Reply:
I actually kept it the same look at line 26

```rust

const STAKING: CurrencyId = GetStableCurrencyId::get();
```

I can change it to staking cause yeah using stablecoin seems a bit nonsensical

Reply:
oh ok that must be a mistake 😓